config/routes.rb

```diff
@@ -5,4 +5,5 @@
   devise_for :teachers
   get 'home/index'
   root 'home#index'
+  mathjax 'mathjax'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.